phpdoc.xml.dist

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You accidentally used php**doc**.xml.dist instead of php**unit**.xml.dist :)

Reply:
Oh damn, silly me, good catch!